boilerPlate/src/app/shared/directives/title-case.directive.ts

```diff
@@ -0,0 +1,16 @@
+import { Directive, ElementRef, HostListener } from '@angular/core';
```

Comment:
Remove this, We don't need this directive as Angular provide by default https://angular.io/api/common/TitleCasePipe

Reply:
done